plonk-neon/src/lib.rs

```diff
@@ -0,0 +1,24 @@
+//! The `Marlin_plonk_stubs` crate exports some functionalities
```

Comment:
This is incorrect. Can you update the description, please?

Related to [this comment](Same comment than [here](https://github.com/o1-labs/proof-systems/pull/3217/files#r2079218001).

Reply:
Agreed for the cleanup in a follow-up. I've been wanting to do it since I moved it here.